modules/collabora_online_group/tests/src/Traits/GroupRelationTrait.php

```diff
@@ -43,4 +43,14 @@ protected function createPluginRelation(GroupTypeInterface $group_type, string $
     return $entity;
   }
 
+  /**
+   * Checks if 'group_relationship_type' group type entity exists.
```

Comment:
First, this is not about "... entity exists" but "... entity type exists".
Second, I think we need to explain why we do this check.
My proposal is just a stub, it could be completely wrong, please correct it.
```suggestion
  /**
   * Checks if the 'group_relationship_type' entity type exists.
   *
   * That config entity type was introduced in group module version xyz....
   * In other versions of group module, the *** entity type was used instead.
```